src/main/webapp/GeppettoConfiguration_production.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
No really. As json doesn't support comments I added the production 'template' as another file. I was not sure if it could be interesting to provide this template... we can delete it if we prefer...